core/src/stores/postgres.rs

```diff
@@ -53,6 +53,13 @@ impl PostgresStore {
             }
         }
 
+        for f in SQL_FUNCTIONS {
+            match c.execute(f, &[]).await {
```

Comment:
I think this is just equivalent to `c.execute(..).await?;` no ? Don't think you need a match

Reply:
For this part I just did the same as for the tables and indexes TBH 🙈